<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>memory</class>
 <widget class="QWidget" name="memory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/static/images/bubu.ico</normaloff>:/static/images/bubu.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget  {
	border:1px solid black;
	border-radius:5px;
}</string>
  </property>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#titleWidget{
background-color:rgb(22, 166, 243);
color:black;
border-radius:5px;
border:1px solid black;
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>单词本</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>1</y>
      <width>39</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover { 
 background-color: red; 
 color: rgb(255, 255, 255);
border: 1px solid black;
font-size:36px;
}
QPushButton {background-color:#E1E1E1;
color:black;
background-color: rgb(22, 166, 243); 
border-radius:5px;
border:0px solid black;
font-size:36px;}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="wordLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>word</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="meanLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>311</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>单词</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="notBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>不会</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>会</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>下一个</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>41</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton:hover { 
color:rgb(0, 0, 0);
border:0px solid black;
background-color:rgb(240,240,240);
}
QPushButton {
color:rgb(149, 149, 149);
border:0px solid black;
background-color:rgb(240,240,240);
}
</string>
   </property>
   <property name="text">
    <string>&lt; 返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover { 
 background-color: red; 
 color: rgb(255, 255, 255);
border: 1px solid black;
	font: 36pt &quot;Microsoft YaHei UI&quot;;
}
QPushButton {background-color:#E1E1E1;
color:black;
background-color: rgb(22, 166, 243); 
border-radius:5px;
border:0px solid black;
	font: 36pt &quot;Microsoft YaHei UI&quot;;}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <zorder>nextBtn</zorder>
  <zorder>titleWidget</zorder>
  <zorder>wordLabel</zorder>
  <zorder>meanLabel</zorder>
  <zorder>notBtn</zorder>
  <zorder>okBtn</zorder>
  <zorder>backBtn</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
